src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor.cs

```diff
@@ -25,31 +25,40 @@ public partial class StateColumnDisplay
     [Inject]
     public required IStringLocalizer<Columns> Loc { get; init; }
 
+    internal static string? GetResourceStateTooltip(ResourceViewModel resource, IStringLocalizer<Columns> loc)
+    {
+        return GetResourceStateTooltip(
+            resource,
+            loc[Columns.StateColumnResourceExitedUnexpectedly].Value,
+            loc[Columns.StateColumnResourceExited].Value,
+            loc[nameof(Columns.RunningAndUnhealthyResourceStateToolTip)]);
```

Comment:
This looks wrong. Actually, the bug existed before, but we can fix it here.

Without the `nameof` this will only ever return the neutral (`en-US`) value:

```suggestion
        return GetResourceStateTooltip(
            resource,
            loc[nameof(Columns.StateColumnResourceExitedUnexpectedly)],
            loc[nameof(Columns.StateColumnResourceExited)],
            loc[nameof(Columns.RunningAndUnhealthyResourceStateToolTip)]);
```

This would make a good analyzer.

Reply:
filed https://github.com/dotnet/aspire/issues/6352